src/Expression.php

```diff
@@ -383,6 +383,26 @@ public function getDebugQuery()
         return $d.' <span style="color:gray">[' . implode(', ', $pp) . ']</span>';
     }
 
+    function __debugInfo()
+    {
+
+        $arr = [
+            'R'=>false,
+            'template'=>$this->template,
+            'params'=>$this->params,
+            'connection'=>$this->connection,
+            'args'=>$this->args,
+        ];
+
+        try {
+            $arr['R'] = $this->render();
+        } catch (\Exception $e) {
```

Comment:
hehe. ok. but leave as-is. we do not know where that information will end up. It's unsafe to expose Params.